gcloud-java-compute/src/main/java/com/google/gcloud/compute/Operation.java

```diff
@@ -0,0 +1,793 @@
+   */
+  public String targetLink() {
+    return targetLink;
+  }
+
+  /**
+   * Returns the unique service-defined target ID, which identifies the resource that the operation
+   * is modifying.
+   */
+  public String targetId() {
+    return targetId;
+  }
+
+  /**
+   * Returns the status of the operation.
+   */
+  public Status status() {
+    return status;
+  }
+
+  /**
+   * Returns an optional textual description of the current status of the operation.
+   */
+  public String statusMessage() {
+    return statusMessage;
+  }
+
+  /**
+   * Returns the user who requested the operation, for example: {@code user@example.com}.
+   */
+  public String user() {
+    return user;
+  }
+
+  /**
+   * Returns an optional progress indicator that ranges from 0 to 100. There is no requirement that
+   * this be linear or support any granularity of operations. This should not be used to guess when
+   * the operation will be complete. This number should monotonically increase as the operation
+   * progresses.
+   */
+  public Integer progress() {
+    return progress;
+  }
+
+  /**
+   * Returns the time that this operation was requested. In milliseconds since epoch.
+   */
+  public Long insertTime() {
+    return insertTime;
+  }
+
+  /**
+   * Returns the time that this operation was started by the service. In milliseconds since epoch.
+   */
+  public Long startTime() {
+    return startTime;
+  }
+
+  /**
+   * Returns the time that this operation was completed. In milliseconds since epoch.
+   */
+  public Long endTime() {
+    return endTime;
+  }
+
+  /**
+   * Returns the errors encountered while processing this operation, if any. Returns {@code null} if
+   * no error occurred.
+   */
+  public List<OperationError> errors() {
+    return errors;
+  }
+
+  /**
+   * Returns the warnings encountered while processing this operation, if any. Returns {@code null}
+   * if no warning occurred.
+   */
+  public List<OperationWarning> warnings() {
+    return warnings;
+  }
+
+  /**
+   * Returns the HTTP error status code that was returned, if the operation failed. For example, a
+   * {@code 404} means the resource was not found.
+   */
+  public Integer httpErrorStatusCode() {
+    return httpErrorStatusCode;
+  }
+
+  /**
+   * Returns the the HTTP error message that was returned, if the operation failed. For example, a
+   * {@code NOT FOUND} message is returned if the resource was not found.
+   */
+  public String httpErrorMessage() {
+    return httpErrorMessage;
+  }
+
+  /**
+   * Returns an optional textual description of the operation.
+   */
+  public String description() {
+    return description;
+  }
+
+  /**
+   * Checks if this operation exists.
+   *
+   * @return {@code true} if this operation exists, {@code false} otherwise
+   * @throws ComputeException upon failure
+   */
+  public boolean exists() throws ComputeException {
+    return reload(Compute.OperationOption.fields()) != null;
```

Comment:
Ah, good to know. Do you think it is worth mentioning in the `delete(OperationId)` javadoc?